Marlin/language_en.h

```diff
@@ -490,5 +490,9 @@
 #ifndef MSG_DELTA_CALIBRATE_CENTER
   #define MSG_DELTA_CALIBRATE_CENTER          "Calibrate Center"
 #endif
+#ifndef MSG_RESET_BLT
+  #define MSG_RESET_BLT                        "Reset BLTouch alarm"  
+#endif
```

Comment:
It's friendly to add the new define to all the other language files as well. The custodians of those files are watching for changes so they can update them.